addons/api/mirage/factories/account.js

```diff
@@ -32,25 +32,28 @@ export default factory.extend({
       case TYPE_AUTH_METHOD_PASSWORD:
         return { login_name: faker.internet.userName() };
       case TYPE_AUTH_METHOD_LDAP: {
-        const groupsAmount = faker.datatype.number({ min: 3, max: 5 });
-        let groups = [];
-        for (let i = 0; i < groupsAmount; i++) {
-          groups.push(faker.random.word());
+        const randomNum = faker.number.int({ min: 3, max: 5 });
+        let member_of_groups = [];
+        const login_name = faker.internet.userName();
+        let dn = `uid=${login_name}`;
+        for (let i = 0; i < randomNum; i++) {
+          member_of_groups.push(faker.word.noun());
+          dn += `,dc=${faker.internet.domainSuffix()}`;
```

Comment:
None blocking comment: Can you elaborate the change in `dn`? before we were returning a string. Now we set it with an `uid` string, and later on the loop we concat a string per iteration. What is the purpose of it?

Reply:
I assume it's to make a more accurate `dn` [field](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/accounts#dn), take a look at what they can look like [here](https://learn.microsoft.com/en-us/previous-versions/windows/desktop/ldap/distinguished-names).

She's setting an initial UID and a random number of DC fields as part of the string